CHANGELOG.md

```diff
@@ -10,11 +10,15 @@
 * Add support for feature gates to User and Topic Operators
 * Add support for setting `publishNotReadyAddresses` on services for listener types other than internal.
 * Update HTTP bridge to latest 0.29.0 release
+* Add support for custom Cruise Control API users
 
 ### Changes, deprecations and removals
 
 * The `reconciliationIntervalSeconds` configuration for the Topic and User Operators is deprecated, and will be removed when upgrading schemas to v1.
   Use `reconciliationIntervalMs` converting the value to milliseconds.
+* Location of Topic Operator API credentials for Cruise Control has changed from
+  `/etc/eto-cc-api/topic-operator.apiAdminUsername` -> `/etc/eto-cc-api/topic-operator.username` for username
+  `/etc/eto-cc-api/topic-operator.apiAdminPassword` -> `/etc/eto-cc-api/topic-operator.password` for password.
```

Comment:
Why are we changing this? What is the motivation or reason?

Reply:
To standardize the username and password file naming for all CC API users in a way that better describes each API user's purpose.

For example in this PR:
 
The file that holds the password for the RebalanceOperator CC API user was changed:
`cruise-control.apiAdminPassword` -> `rebalance-operator.password`

The file that holds the password  for the API user that checks if the CC API is up and running was changed 
`cruise-control.apiUserPassword` -> `healthcheck.password`.

Lastly to follow the pattern:

The file that holds the username for the TopicOperator CC API user was changed:

 `topic-operator.apiAdminUsername` -> `topic-operator.username`

The file that holds the password for the TopicOperator CC API password was changed:

 `topic-operator.apiAdminPassword` -> `topic-operator.password`

Reply:
But this change affects users, so is it really worth it?

Reply:
> But this change affects users, so is it really worth it?

People using the standalone TO with Cruise Control would need to be aware of this filename change and update their TO  deployments accordingly. Not a huge price to pay for standardized naming but understand how the change could be annoying. 

If you feel strongly about it, I'll revert the change

Reply:
I would not feel strongly about it if it had some good reason. But it seems it is just a change for a sake of change. What are you standardizing? There is only one TO with these names. They don't relate to anything else, or?

Reply:
> What are you standardizing?

The name of the keys for the CC API usernames and passwords in the CC API secrets (and filenames when these CC API secrets are mounted into pod) used by the Topic Operator (TO) and Rebalance Operator (RO) 

To have the TO API secret and the RO API secret share the a key naming format. Right now the key names have the CC API role incorporated as part of the name which is unnecessary

Reply:
@kyguy I would also prefer to revert this change TBH. An alternative, as suggested before, could be to deprecate the old keys logging a warning when used, and then remove them in a couple of releases. Wdyt?

Reply:
@fvaleri But you would need to do that in the TO I guess and check if the files are or aren't there and use an alternative one? The CC feature in TO is relatively new, so I'm not sure it is worth the deprecation and removal later if we want to use the new names long term.

@ppatierno Any thoughts on this?

Reply:
@kyguy The TO Secret is IIRC owned by the TO. So why can't it use its names? The names of the other secret used by the CO matter less as they are not user-facing.

Reply:
> The CC feature in TO is relatively new, so I'm not sure it is worth the deprecation and removal later if we want to use the new names long term.

I was just proposing an alternative, but I feel that the motivation for this change is not strong enough, so my vote is on reverting the change.

Reply:
I am for having better and consistent naming so I would like this change but ...

> People using the standalone TO with Cruise Control would need to be aware of this filename change and update their TO deployments accordingly

How many users are really using the TO with Cruise Control in standalone mode which is a pretty new thing. AFAIU it's going to be just a matter of describing the "breaking" change in the CHANGELOG.
Of course, deprecating and remove could be the way for this kind of things but again, I don't think many people are using it.

Reply:
> The TO Secret is IIRC owned by the TO. So why can't it use its names? 

It is, and it can use the existing names no problem. 

> The names of the other secret used by the CO matter less as they are not user-facing.

True. While I was updating the key names in to CO API secret I figured I would update the key names in the TO API secret while I was in there. However, it does not seem worth it since the key names of the TO API secret are user facing. TBH I don't want to complicate things or delay this PR any longer. If it is ok with everyone, I will revert the TO API secret key names to how they were